lib/chat_api_web/controllers/note_controller.ex

```diff
@@ -0,0 +1,65 @@
+defmodule ChatApiWeb.NoteController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.Notes
+  alias ChatApi.Notes.Note
+
+  action_fallback ChatApiWeb.FallbackController
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, %{"note" => note_params}) do
```

Comment:
for the params, i think we can simplify it to just `%{"customer_id" => customer_id}` (rather than `%{"note" => "customer_id" => customer_id}}`)

then the API request would just look like
```
GET /api/notes?customer_id=xxxxx
```